README.md

```diff
@@ -3082,15 +3082,30 @@ The test file pattern and exclude pattern support any glob pattern handled by [`
 
 #### How to run a specific list of test files or only some tests from test file?
 
-:information_source: If you don't want to use the pattern [`KNAPSACK_PRO_TEST_FILE_PATTERN`](#how-can-i-run-tests-from-multiple-directories) to define a list of tests to run then read below.
+:information_source: If you don't want to use the pattern [`KNAPSACK_PRO_TEST_FILE_PATTERN`](#how-can-i-run-tests-from-multiple-directories) to define a list of tests to run then read below two options.
+
+**Option 1:**  
 
 If you want to run a specific list of test files that are explicitly defined by you or auto-generated by some kind of script you created then please use:
 
 `KNAPSACK_PRO_TEST_FILE_LIST=spec/features/dashboard_spec.rb,spec/models/user.rb:10,spec/models/user.rb:29`
 
 Note `KNAPSACK_PRO_TEST_FILE_LIST` must be a list of test files comma separated. You can provide line number for tests inside of spec file in case of RSpec (this way you can run only one test or a group of tests from RSpec spec file). You can provide the same file a few times with different test line number.
 
-Note when you set `KNAPSACK_PRO_TEST_FILE_LIST` then below environment variables are ignored:
+**Option 2:** 
+
+Similarly, you can also provide a source file containing the test files that you would like to run. For example:
+`KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE=spec/fixtures/test_file_list_source_file.txt`  
+And the content of the test file can be any format below or combination:
+```
+./spec/test1.rb,
```

Comment:
@Neal910 I worry that this format might be confusing for users (having option to use comma and/or new line).

Wouldn't be more clear if we would recommend using always each test file in a new line? And then we don't need to handle removing `,` commas.

**Is there a reason why would you like to see support for removing commas?** 

If you think we should support both (new line and comma) then maybe it would be better to have 2 example lists here instead of mixed examples to make it more readable for users.

Reply:
I checked the other project's output , it has the comma in the end of each line. I was using a `sed` command to remove the comma, then I thought maybe I should add this feature into the Knapsack as well so I don't have to handle it using a separate command. I am not sure if you want to handle the comma or not.

Reply:
It's true that it will be much clear to only handle the new line scenario. If you made the call, I can remove it. Then I will just keep using the `sed` to change it.

Reply:
Ok. Let's keep it simple and just use newlines.

It's easy to make too many features into the gem and then it might be confusing for users or hard for maintenance in the future. Simplicity might be good.